docs/rules/0148/use-uid.md

```diff
@@ -0,0 +1,63 @@
+---
+rule:
+  aip: 148
+  name: [core, '0148', use-uid]
+  summary: Use uid instead of id in resource messages.
+permalink: /148/use-uid
+redirect_from:
+  - /0148/use-uid
+---
+
+# Human names
+
+This rule encourages the use of `uid` instead of `id` for resource messages, as
+mandated in [AIP-148][].
+
+## Details
+
+This rule looks for fields named `first_name` and `last_name`, and complains if
```

Comment:
looks like this should be changed?

Reply:
Ha yeah, forgot to update this completely. Done.